### ISSUE TITLE
3.28.3 The units of equations do not affect the maths

### ISSUE THREAD

**Issue by MichaelClerx** [cellml/cellml-specification#326]
https://github.com/cellml/cellml-specification/blob/cf2bf5c7897b6e147128522c19c1a6d0f33fcba6/src/reference/sectionC_interpretation.inc#L339

In general, we've taken the approach that:

1. The validity of the mathematics does not affect the validity of the CellML document
2. The CellML specification only concerns itself with the validity of the CellML document

and so we shouldn't really be telling the user how to interpret the mathematics?

**Comment by kerimoyle** [comment]
I don't quite understand the problem?  I took "mathematical interpretation" to mean "mathematical interpretation by CellML" not by the modellers ... is that what you're talking about? I guess we could reword it to be "... SHALL NOT affect the validity of mathematical statements in the CellML model." or "The validity of a CellML model SHALL NOT be affected by units referenced by a :code:`units` attribute." or something?

**Comment by MichaelClerx** [comment]
No no it's nothing like that. We explicitly removed all things like "You can't have x = 1; x = 2", or "you can't have initial_value(x)=3; x = 4". We've removed all rules that depended on the interpretation of the mathematics.

So why do we still have this one?
